### ISSUE TITLE
Star Ocean Second Story R crash when leaving arlia

### ISSUE THREAD

**Issue by MrStu82** [yuzu-emu/yuzu#12027]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Android build 131

### Description of Issue

Near the beginning of the game, I leave arlia for the next town. The screen goes to the normal loading screen with the hopping animal in the lower right. Then the animal freezes, like it's about to display the new town, but it crashes back out to the game select menu.

### Expected Behavior

I expect it to finish the loading animation, then display the town I'm moving to.

### Reproduction Steps

Leave Arlia Town. It's near the beginning of the game.

### Log File

https://drive.google.com/file/d/1d0iCwAftHTtnZjNd3PcrI1isVleHGfZI/view?usp=drivesdk

### System Configuration

CPU:
GPU/Driver:
RAM:
OS:

Android 13
Sony Xperia 1 V
Snapdragon 8 gen 2
Adreno 740

**Comment by Vespars** [comment]
This crash is actually something that can happen on any major zoning (city to world or to dungeon). It happens sometimes but not most of the time for me.
[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/13467763/yuzu_log.txt.old.txt)

**Comment by HxBreak** [comment]
I am not encountered this crash?maybe it 's already be fixed?

**Comment by MrStu82** [comment]
> I am not encountered this crash?maybe it 's already be fixed?

No, still happening in the latest build. Also tried it on my new Odin 2, still happening. Also tried updating to the new turnip v24 rev11 driver's, still happening

**Comment by HxBreak** [comment]
looks like fault on snapdragon soc, I am using mediatek 9200, only found resource leak&cg video audio not sync problem on this game
